examples/nginx-access-logs-streaming/README.md

```diff
@@ -0,0 +1,217 @@
+
+<details>
+<summary>nginx logs directory listing.</summary>
+<pre>
+ll /var/log/nginx/
+total 2152
+drwxr-xr-x  3 root     adm      4096 Apr 30 00:00 ./
+drwxrwxr-x 13 root     syslog   4096 Apr 28 00:00 ../
+drwxr-xr-x  2 www-data adm      4096 Apr 28 16:42 2020-10_2023-06/
+-rw-r-----  1 www-data adm    418174 Apr 30 22:36 access.log
+-rw-r-----  1 www-data adm    799201 Apr 29 23:59 access.log.1
+-rw-r-----  1 www-data adm     47009 Apr 20 23:59 access.log.10.gz
+-rw-r-----  1 www-data adm     49434 Apr 19 23:59 access.log.11.gz
+-rw-r-----  1 www-data adm     44373 Apr 19 00:00 access.log.12.gz
+-rw-r-----  1 www-data adm    109735 Apr 17 23:59 access.log.13.gz
+-rw-r-----  1 www-data adm     64309 Apr 17 00:00 access.log.14.gz
+-rw-r-----  1 www-data adm     41989 Apr 28 23:59 access.log.2.gz
+-rw-r-----  1 www-data adm     40176 Apr 27 23:59 access.log.3.gz
+-rw-r-----  1 www-data adm     51326 Apr 26 23:59 access.log.4.gz
+-rw-r-----  1 www-data adm     96012 Apr 26 00:00 access.log.5.gz
+-rw-r-----  1 www-data adm     57779 Apr 24 23:59 access.log.6.gz
+-rw-r-----  1 www-data adm     54235 Apr 23 23:59 access.log.7.gz
+-rw-r-----  1 www-data adm     91835 Apr 22 23:59 access.log.8.gz
+-rw-r-----  1 www-data adm     89026 Apr 22 00:00 access.log.9.gz
+-rw-r-----  1 www-data adm      4080 Apr 30 00:28 error.log
+-rw-r-----  1 www-data adm     33801 Apr 29 23:35 error.log.1
+-rw-r-----  1 www-data adm       221 Apr 18 16:02 error.log.10.gz
+-rw-r-----  1 www-data adm       535 Apr 17 21:11 error.log.11.gz
+-rw-r-----  1 www-data adm       224 Apr 16 23:01 error.log.12.gz
+-rw-r-----  1 www-data adm       227 Apr 15 06:32 error.log.13.gz
+-rw-r-----  1 www-data adm       331 Apr 14 15:44 error.log.14.gz
+-rw-r-----  1 www-data adm       313 Apr 28 05:53 error.log.2.gz
+-rw-r-----  1 www-data adm       274 Apr 27 11:49 error.log.3.gz
+-rw-r-----  1 www-data adm       319 Apr 26 09:45 error.log.4.gz
+-rw-r-----  1 www-data adm       279 Apr 25 13:30 error.log.5.gz
+-rw-r-----  1 www-data adm       350 Apr 23 22:43 error.log.6.gz
+-rw-r-----  1 www-data adm       291 Apr 22 22:27 error.log.7.gz
+-rw-r-----  1 www-data adm       265 Apr 21 05:55 error.log.8.gz
+-rw-r-----  1 www-data adm       292 Apr 19 21:49 error.log.9.gz
+</pre>
+</details>
+
+In addition to log file rotation, nginx uses compression to keep the storage requirements of log files inside `/var/log/nginx/` to a minimum. After about 4 years, the `/var/log/nginx/` weighs in at just `2.7MB`:
+```bash
+du -shL /var/log/nginx/
+2.7M	/var/log/nginx/
+```
+
+The Node.js web app for Ghost on the other hand writes its logs to `/var/www/ghost/content/logs/` and that folder is already approaching `700MB` on disk: 
+```bash
+du -shL /var/www/ghost/content/logs/
+692M  /var/www/ghost/content/logs/
+```
+To keep things simple, I'll only make use of the access logs for nginx in the article.
```

Comment:
Processing 700MB logs is certainly more impressive than 2.7MB. Maybe you just skip the mentioning of compressed files